SIDFactoryII/config.ini

```diff
@@ -33,6 +33,8 @@ Sound.Emulation.Resample            = 1         // If this is set to 1, the SID
 
 Sound.Output.Gain                   = 1.0       // Boost/lower volume. Sound can become distorted for value higher than 1.0.
 
+Sound.Emulation.SampleFrequency		= 44100     // Output sample frequency in Hz (22050/44100/48000/96000 etc.)
```

Comment:
Could you copy this to `dist/documentation/user.default.ini`?

It is a double administration for now, sorry.

Reply:
Also done.